crates/bevy_ecs/src/system/commands/command_queue.rs

```diff
@@ -43,45 +43,47 @@ impl CommandQueue {
     where
         C: Command,
     {
-        let old_len = self.bytes.len();
+        #[repr(C, packed)]
```

Comment:
This needs a comment as to why we need  `repr(packed)` here. Preferably with a link to the appropriate Rustinomicon page.

Reply:
> This needs a comment as to why we need `repr(packed)` here.

Good point

> Probably also a safety comment too as `repr(packed)` can make UB from safe code.

Rustc currently has a [deny-by-default lint](https://github.com/rust-lang/rust/issues/82523) for that unsoundness, which will later get promoted into a hard error. IMO we can get away with just not worrying about it -- if this code compiles, then we know it's safe.